<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwind</class>
 <widget class="QMainWindow" name="mainwind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>538</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>562</width>
    <height>433</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mainwind</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 230, 230)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="sepiaWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="sepia_2">
     <item>
      <widget class="QLabel" name="depthLabel">
       <property name="text">
        <string>Depth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="depthSlider">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>25</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="intensityLabel">
       <property name="text">
        <string>Intensity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="intensitySlider">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>25</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sepiaSave">
       <property name="styleSheet">
        <string notr="true">white</string>
       </property>
       <property name="text">
        <string>Commit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="brightnessWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="brightLayout">
     <item>
      <widget class="QLabel" name="brightLabel">
       <property name="text">
        <string>Brightness</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="brightSlider">
       <property name="minimum">
        <number>-50</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="brightSave">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>Commit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>15</y>
      <width>431</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File </string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRevert"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionRotateRight"/>
    <addaction name="actionRotateLeft"/>
    <addaction name="actionRotate_180"/>
    <addaction name="actionFlip_2"/>
    <addaction name="actionFilters_3"/>
    <addaction name="separator"/>
    <addaction name="actionTools"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Photo_Editor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230)</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionSave"/>
   <addaction name="actionFilters_3"/>
   <addaction name="actionTools"/>
   <addaction name="actionAbout_Photo_Editor"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="StickerWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 210, 210)</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="StickerWidgetContents">
    <widget class="QToolButton" name="flip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>42</width>
       <height>42</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="imgs.qrc">
       <normaloff>:/new/prefix1/images copy/flip-horizontally-design-graphic-tool-3bd1aae13ed68d2f-512x512.png</normaloff>:/new/prefix1/images copy/flip-horizontally-design-graphic-tool-3bd1aae13ed68d2f-512x512.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="rotate180">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>42</width>
       <height>42</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="imgs.qrc">
       <normaloff>:/new/prefix1/images copy/rotation-512.png</normaloff>:/new/prefix1/images copy/rotation-512.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="undo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>42</width>
       <height>42</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="imgs.qrc">
       <normaloff>:/new/prefix1/images copy/button_undo-128.png</normaloff>:/new/prefix1/images copy/button_undo-128.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="rotate_left">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>42</width>
       <height>42</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="imgs.qrc">
       <normaloff>:/new/prefix1/images copy/arrow13-512-1.png</normaloff>:/new/prefix1/images copy/arrow13-512-1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="rotate_right">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>42</width>
       <height>42</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="imgs.qrc">
       <normaloff>:/new/prefix1/images copy/arrow13-512.png</normaloff>:/new/prefix1/images copy/arrow13-512.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="revert">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="imgs.qrc">
       <normaloff>:/new/prefix1/images copy/download copy.png</normaloff>:/new/prefix1/images copy/download copy.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="FilterWidget">
   <property name="minimumSize">
    <size>
     <width>144</width>
     <height>340</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 210, 210)</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="FilterWidgetContents">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>0</y>
       <width>140</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="blackAndWhite">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Black and White</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="grayscale">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Grayscale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="aqua">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Aqua</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rouge">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Rouge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="veridian">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Veridian</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="negative">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Negative</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sepia">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Sepia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="brightness">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Brightness</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/Actions-folder-new-icon.png</normaloff>:/new/prefix1/images copy/Actions-folder-new-icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/Open-file-icon copy.png</normaloff>:/new/prefix1/images copy/Open-file-icon copy.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/save-icon-28.png</normaloff>:/new/prefix1/images copy/save-icon-28.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/quit-512.png</normaloff>:/new/prefix1/images copy/quit-512.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/stock_cut.png</normaloff>:/new/prefix1/images copy/stock_cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/qemytmkyvcboqlglolrs.png</normaloff>:/new/prefix1/images copy/qemytmkyvcboqlglolrs.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionRotate_Right">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/arrow13-512.png</normaloff>:/new/prefix1/images copy/arrow13-512.png</iconset>
   </property>
   <property name="text">
    <string>Rotate Right</string>
   </property>
  </action>
  <action name="actionFlip">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/flip-horizontally-design-graphic-tool-3bd1aae13ed68d2f-512x512.png</normaloff>:/new/prefix1/images copy/flip-horizontally-design-graphic-tool-3bd1aae13ed68d2f-512x512.png</iconset>
   </property>
   <property name="text">
    <string>Flip</string>
   </property>
  </action>
  <action name="actionFilters">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/291378-language-512.png</normaloff>:/new/prefix1/images copy/291378-language-512.png</iconset>
   </property>
   <property name="text">
    <string>Add Text</string>
   </property>
  </action>
  <action name="actionFilters_2">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/Filter-512.png</normaloff>:/new/prefix1/images copy/Filter-512.png</iconset>
   </property>
   <property name="text">
    <string>Filters</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/paste.png</normaloff>:/new/prefix1/images copy/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/button_undo-128.png</normaloff>:/new/prefix1/images copy/button_undo-128.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/button_redo-512.png</normaloff>:/new/prefix1/images copy/button_redo-512.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCrop_2">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/cut.png</normaloff>:/new/prefix1/images copy/cut.png</iconset>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionRotateRight">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/arrow13-512.png</normaloff>:/new/prefix1/images copy/arrow13-512.png</iconset>
   </property>
   <property name="text">
    <string>Rotate Right</string>
   </property>
  </action>
  <action name="actionRotateLeft">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/arrow13-512-1.png</normaloff>:/new/prefix1/images copy/arrow13-512-1.png</iconset>
   </property>
   <property name="text">
    <string>Rotate Left</string>
   </property>
  </action>
  <action name="actionFlip_2">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/flip-horizontally-design-graphic-tool-3bd1aae13ed68d2f-512x512.png</normaloff>:/new/prefix1/images copy/flip-horizontally-design-graphic-tool-3bd1aae13ed68d2f-512x512.png</iconset>
   </property>
   <property name="text">
    <string>Flip</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/Actions-folder-new-icon.png</normaloff>:/new/prefix1/images copy/Actions-folder-new-icon.png</iconset>
   </property>
   <property name="text">
    <string>Add Text</string>
   </property>
  </action>
  <action name="actionFilters_3">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/Filter-512.png</normaloff>:/new/prefix1/images copy/Filter-512.png</iconset>
   </property>
   <property name="text">
    <string>Filters</string>
   </property>
  </action>
  <action name="actionTools">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/wrench-512.png</normaloff>:/new/prefix1/images copy/wrench-512.png</iconset>
   </property>
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionAbout_Photo_Editor">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/info-512.png</normaloff>:/new/prefix1/images copy/info-512.png</iconset>
   </property>
   <property name="text">
    <string>About Photo Editor</string>
   </property>
  </action>
  <action name="actionRotate_180">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/new/prefix1/images copy/rotation-512.png</normaloff>:/new/prefix1/images copy/rotation-512.png</iconset>
   </property>
   <property name="text">
    <string>Rotate 180</string>
   </property>
  </action>
  <action name="actionRevert">
   <property name="text">
    <string>Revert</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
